### DISCUSSION TITLE
Is my connectiion being rerouted through invidious instance?

### DISCUSSION THREAD

**Discussion by Raj2032** [FreeTubeApp/FreeTube#4106] [Q&A] [answered]
![image](https://github.com/FreeTubeApp/FreeTube/assets/29293830/75d4d605-ceb9-408a-85e9-ce92880189da)

I have this on so does this mean that videos will be re-routed through invidious so that Google doesn't get my IP while watching videos?

**Comment by ChunkyProgrammer** [comment, marked as answer]
You'd probably want to set `Preferred API Backend` to Invidious, Disable `Fallback to Non-Preferred Backend On Failiure` and Enable `Proxy Videos through Invidious`

**Reply by Raj2032** [reply]
> and Enable Proxy Videos through Invidious

Ok so over here:

![image](https://github.com/FreeTubeApp/FreeTube/assets/29293830/d0738715-50b8-4bb1-beea-975fc34d8968)

is this the correct settings?

**Reply by ChunkyProgrammer** [reply]
Proxy Videos Through Invidious is under `Player Settings` 🙂
